core/DataTable.php

```diff
@@ -1894,7 +1894,7 @@ protected function aggregateRowWithLabel(Row $row, $columnAggregationOps)
     {
         $labelToLookFor = $row->getColumn('label');
         if ($labelToLookFor === false) {
-            throw new Exception("Label column not found in the table to add in addDataTable()");
+            throw new Exception("Label column not found in the table to add in addDataTable() for row " . var_export($row, 1));
```

Comment:
This happened in the GA importer, when debugging I found  it would be more useful to have archiver params here, ie idSite/period/date and maybe report name. What do you think?

Reply:
That sounds good @diosmosis 

btw is that bug issue fixed in the importer? I think the customer might have actually the issue with the importer

Reply:
If it's the same bug, then yes, it was an issue in the Referrers.getWebsites report, one of them will have a blank label causing period aggregation to fail this way. Can be fixed by re-importing the specific day(s).

Do you want to add the debug code or should I?